src/dom/dimensions-2d-dom.ts

```diff
@@ -0,0 +1,21 @@
+export class Dimensions2dDom {
```

Comment:
I assume this is the start of the class?  As is I don't feel we should check this in as it doesn't add too much value but I can see is building more methods to this to make it a useful tool.

Reply:
I think the value is in readability w/o additional comments versus actual functionality.

```
  const dimensions = Dimensions2dDom.fromElementOffset(this.el);
  if (this.lastDimensions && dimensions.equals(this.lastDimensions)) {
    return;
  }
  ```

  Reads clearer to me than...
  
  ```
  const dimensions = [this.el.offsetWidth, this.el.offsetHeight];
  if (
      this.lastDimensions &&
      this.lastDimensions[0] === dimensions[0] &&
      this.lastDimensionsp[1] === dimensions[1]
  ) {
    return;
  }
  ```
  
  That said I can solve this with comments, so I will remove this for now.